### PR TITLE
fix(gate) - trigger orders updates

### PR DESCRIPTION
**Pull request by ttodua**
this is similar update and should also be merged: https://github.com/ccxt/ccxt/pull/23694

### CODE REVIEW

File: ts/src/test/static/request/gate.json [hunk 1]
```diff
@@ -392,6 +392,20 @@
                     "LTC/USDT"
                 ]
             },
+            {
```
Comment (carlosmiei) [inline ts/src/test/static/request/gate.json:395]: is it possible to add swap+trigger tests?

Comment (ttodua) [reply ts/src/test/static/request/gate.json:395]: > is it possible to add swap+trigger tests?

pushed atm

Comment (carlosmiei) [reply ts/src/test/static/request/gate.json:395]: @ttodua hum I can't see them here

Comment (ttodua) [reply ts/src/test/static/request/gate.json:395]: > @ttodua hum I can't see them here

image

i was referring to this one, do you mean smth other?

Comment (carlosmiei) [reply ts/src/test/static/request/gate.json:395]: @ttodua do we have trigger+swap fetchOpenOrders static request tests?

**Pull request merged**